src/test/java/jalau/at18/katas/highestlowest/alvaro/MainTest.java

```diff
@@ -0,0 +1,30 @@
+package jalau.at18.katas.highestlowest.alvaro;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class MainTest {
```

Comment:
There might be a missing test, the one for verifying the main kata goal, get highest and lowest as result. Actually, you did a great job here separating concerns in smaller methods, and that is very good, but you should have a function that calls both and generates the final result that is both values, which is the main requirement of the kata